license-config.json

```diff
@@ -40,7 +40,8 @@
     "packages/dashboards/*.jar",
     "packages/dashboards/plugins/*.jar",
     "packages/dashboards/data/metabase/*.db",
-    "packages/client/dev-dist"
+    "packages/client/dev-dist",
+    "CHANGELOG"
```

Comment:
This shouldn't be required. The issue with the filename was only in the develop branch which is fixed now